docs/draft/6-payloads.md

```diff
@@ -144,7 +150,7 @@ message ChatMessage {
 | 6 | chat_id | `string` | The local ID of the chat the message is sent to |
 | 7 | message_type | `MessageType` | The type of message, different for one-to-one, public or group chats |
 | 8 | content_type | `ContentType` | The type of the content of the message | 
-| 9 | payload | `Sticker|Image|nil` | The payload of the message based on the content type |
+| 9 | payload | `Sticker` I `Image` I `EmojiReaction` I `EmojiReactionRetraction` I `nil` | The payload of the message based on the content type |
```

Comment:
`|` would be parsed by the markdown renderer as a new table cell token, even though it was inside backticks. I've replaced `|` with `I` because markdown font renders `I` as "I"